gatsby-browser.js

```diff
@@ -33,69 +32,6 @@ if (is_browser) {
     if (window.location.pathname.includes(bug_bounty_path)) window.location.href = hacker_one_url
 }
 
-const sendTags = (api) => {
-    const language = LocalStore.get('i18n') || ''
-    const domain = getDomain()
-    const { loginid, residence } = getClientInformation(domain) || {
-        loginid: '',
-        residence: '',
-    }
-    api.getTags()
-        .then((result) => {
-            if (!result || !result.result) {
-                return null
-            }
-            if (
-                !result.result['Login ID'] ||
-                !result.result['Site Language'] ||
-                !result.result.Residence
-            ) {
-                return api.setTags({
-                    'Login ID': loginid,
-                    'Site Language': language.toLowerCase(),
-                    Residence: residence,
-                })
-            }
-            return null
-        })
-        .catch((e) => {
-            // eslint-disable-next-line no-console
-            console.error(e)
-            return null
-        })
-}
```

Comment:
since we were calling this method inside `pushwooshInit` I removed this as well. 

@habib-deriv are we going to have another web-push service in the future or we need this method , please let me know so I can revert it :pray:

Reply:
Hi @shayan-deriv , we'll have Clevertap in the future. That impelementation might be different. So it's better we remove the code.